optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/resources/META-INF/beans.xml

```diff
@@ -14,12 +14,11 @@
   ~ limitations under the License.
   -->
 
-<beans>
-  <scan>
-    <exclude name="**.client.**"/>
```

Comment:
I am not exactly sure how this exclusion worked before, but Weld 2.x refuses to parse the current pattern. In any case, the tests are now working, so I would say the client classes are not leaking anymore.